toot/console.py

```diff
@@ -482,6 +482,12 @@ def editor(value):
                 "help": "plain-text description of the media for accessibility "
                         "purposes, one per attached media"
             }),
+            (["--thumbnail"], {
```

Comment:
Should this `--thumbnail` argument be added to the `toot upload` command as well?

Reply:
Yeah, it probably should. I'll take care of it later.